src/Command/RunCommand.php

```diff
@@ -318,22 +325,12 @@ private function getComponent(ClientWrapper $clientWrapper, string $id): array
     }
 
     /**
-     * @param array<JobDefinition> $jobDefinitions
-     * @return array<JobDefinition>
+     * @return non-empty-string
      */
-    private function resolveVariables(
-        ClientWrapper $clientWrapper,
-        array $jobDefinitions,
-        ?string $branchId,
-        ?string $variableValuesId,
-        array $variableValuesData
-    ): array {
-        if ($variableValuesId === '') {
-            throw new InvalidArgumentException('$variableValuesId must not be empty string');
-        }
-
+    private function resolveBranchId(ClientWrapper $clientWrapper, ?string $branchId): string
+    {
         if ($branchId === null || $branchId === 'default') {
-            $branchesApiClient = new DevBranches($clientWrapper->getBranchClientIfAvailable());
+            $branchesApiClient = new DevBranches($clientWrapper->getBasicClient());
```

Comment:
Rano jsme na standupu zashlechl neoc jako ze branche funguji jen pres nebranchovaneho clienta? Radeji jsme to teda jeste prehodil.